app/helpers/admin/taggable_content_helper.rb

```diff
@@ -101,7 +101,7 @@ def taggable_statistical_data_sets_container
   # of the array consists of two values: the location name and its ID
   def taggable_world_locations_container
     Rails.cache.fetch(taggable_world_locations_cache_digest, expires_in: 1.day) do
-      WorldLocation.ordered_by_name.map {|w| [w.name, w.id] }
+      WorldLocation.ordered_by_name.select(&:active).map { |w| [w.name, w.id] }
```

Comment:
Should we make this a `where(active: true)`? Although I'm guessing there aren't many `WorldLocation` records to load into memory.